test/boost/database_test.cc

```diff
@@ -596,9 +596,10 @@ future<> do_with_some_data(std::vector<sstring> cf_names, std::function<future<>
                                                   cf_name))
                         .get();
                     f1.get();
-                    e.get_system_keyspace().local().load_built_views().get();
 
+                    auto f2 = e.local_view_builder().wait_until_built("ks", "index_cf_index");
```

Comment:
does `wait_until_built` work for index building as well?

Reply:
Scylla's secondary index is implemented with a materialized view. For the index named "index_cf", the underlying view's name is "index_cf_index". So if you wait for this view to be built, then the index is indeed built.

Reply:
nice, thanks for the explanation @nyh